content/pages/syllabus.md

```diff
@@ -44,4 +44,14 @@ Students will also be given an educational account on the Habanero cluster compu
 
 **Grading:** Weekly assignment (70%), final project (30%)
 
+**Assignment Grading Rubric:** 
+
++ Total: 100
++ Everything done and answers all correct: 80 <br/>
+  * Missing answer: -5 per question <br/>
+  * Wrong answer: -2 per question <br/>
```

Comment:
the problem with assigning specific points here is that it is not guaranteed to add up to 80. What if there are 5 questions and the student does nothing? Then they are penalized 5*(5 + 2) = 35. This does not equal 80!

Reply:
That's true. Last year we actually had 15 ~ 20 questions per assignment, so the total should roughly be 80. I think students will try their best to do every question, so the extreme case that student does nothing is unlikely to happen. If you prefer not specifying points here, we can decide the points for each assignment individually?

Reply:
Ok, so what if we say:

> - All questions complete: 50
> - All questions correct: 30
> (e.g., if there are 10 questions, each questions has 5 points for completeness and 3 points for correctness)

Reply:
Sounds good to me. Thanks!